x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -136,6 +184,20 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         await find.byCssSelector('[data-test-subj*="tags-delete-action"]');
       });
 
+      it('shows error when more than 200 tags are added to the case', async () => {
```

Comment:
I am a bit skeptical about this as it may take too long and become flaky. Though the flaky test runner says it is ok.

Reply:
yeah, this test is tricky for unit tests as well, that's why I thought e2e can handle it. 
Flaky test runner didn't complain so I guess we could give it a try.

Reply:
Ok, let's give it a try.